pkg/stub/imagestreams.go

```diff
@@ -207,8 +207,11 @@ func (h *Handler) upsertImageStream(imagestreamInOperatorImage, imagestreamInClu
 }
 
 func (h *Handler) updateDockerPullSpec(oldies []string, imagestream *imagev1.ImageStream, opcfg *v1.Config) {
-	//TODO remove when TBR creds sorted out
-	if imagestream.Name == "jenkins" {
+	//TODO remove when TBR creds sorted out and we do not explicitly set SamplesRegistry to registry.access.redhat.com
+	// by default we want to leave the jenkins images as using the payload set to the IMAGE* env's
+	// but if the customer overrides (after we switch to TBR and don't set SamplesRegistry by default), we'll let them
+	// change jenkins related images
+	if strings.HasPrefix(imagestream.Name, "jenkins") {
```

Comment:
probably better to call out the 3(?) we explicitly provide, just in case someone upstream contributes a "jenkins-foo" imagestream that we want to install later, but isn't part of the payload.

Reply:
update pushed